### PR TITLE
Display expired sessions

### PR DESCRIPTION
**Pull request by malted**
This PR displays past user sessions in the past 7 days
image

Consistently display dates at the bottom of the card 
image
image
image

Oh also adds a blue border to oauth sessions to match that of the current session

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -365,6 +365,15 @@ def edit_security
                                  .includes(:application)
     @all_sessions = (@sessions + @oauth_authorizations).sort_by { |s| s.created_at }.reverse!
 
+    @expired_sessions = @user
+                        .user_sessions
+                        .only_deleted
+                        .not_impersonated
+                        .where('created_at >= ?', 1.week.ago)
```
Comment (sampoder) [inline app/controllers/users_controller.rb]:
```suggestion
                        .where('last_seen_at >= ?', 1.week.ago)
```
Thoughts?

File: app/views/users/edit_security.html.erb [hunk 2]
```diff
@@ -100,6 +100,24 @@
       <% end %>
     <% end %>
   </div>
+
+  <h3 class="flex items-center">
+    Past sessions
```
Comment (sampoder) [inline app/views/users/edit_security.html.erb]:
```suggestion
    Past sessions
```
We should probably clarify that these are just last week's

File: app/views/users/edit_security.html.erb [hunk 3]
```diff
@@ -100,6 +100,23 @@
...
+    <div class="grid grid--split">
+      <% @expired_sessions.each do |session| %>
+        <% if session.is_a? UserSession %>
+          <%= render "user_session", session: %>
+        <% elsif session.is_a? ApiToken %>
+          <%= render "oauth_authorization", authorization: session %>
+        <% end %>
+      <% end %>
+    </div>
+  <% end %>
```
Comment (sampoder) [inline app/views/users/edit_security.html.erb]:
```suggestion
  <% if @expired_sessions.any? %>
    <h3 class="inline-flex items-center">
      Recently logged out
      <%= badge_for @expired_sessions.count, class: "bg-muted" %>
    </h3>
    <div class="grid grid--split">
      <% @expired_sessions.each do |session| %>
        <% if session.is_a? UserSession %>
          <%= render "user_session", session: %>
        <% elsif session.is_a? ApiToken %>
          <%= render "oauth_authorization", authorization: session %>
        <% end %>
      <% end %>
    </div>
  <% end %>
```

**Review by sampoder — APPROVED** [review]
LGTM!

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -365,6 +365,15 @@ def edit_security
                                  .includes(:application)
     @all_sessions = (@sessions + @oauth_authorizations).sort_by { |s| s.created_at }.reverse!
 
+    @expired_sessions = @user
```
Comment (garyhtou) [inline app/controllers/users_controller.rb]:
```ruby
    @expired_sessions = @user
                        .user_sessions
                        .only_deleted
                        .not_impersonated
                        .where("deleted_at >= ?", 1.week.ago)
                        .or(@user.user_sessions.where("expiration_at < ?", Time.now))
                        .order(created_at: :desc)
```

**Pull request merged**